spec/features/course/assessment/question/voice_response_management_spec.rb

```diff
@@ -0,0 +1,81 @@
+# frozen_string_literal: true
+require 'rails_helper'
+
+RSpec.describe 'Course: Assessments: Questions: Voice Response Management', js: true do
+  let(:instance) { Instance.default }
+
+  with_tenant(:instance) do
+    let(:course) { create(:course) }
+    let(:assessment) { create(:assessment, course: course) }
+    before { login_as(user, scope: :user) }
+
+    context 'As a Course Manager' do
+      let(:user) { create(:course_manager, course: course).user }
+
+      scenario 'I can create a new voice response question' do
+        skill = create(:course_assessment_skill, course: course)
+        visit course_assessment_path(course, assessment)
+        click_on 'New Question'
+        new_voice_page = window_opened_by { click_link 'Audio Response' }
+
+        within_window new_voice_page do
+          question_attributes = attributes_for(:course_assessment_question_voice_response)
+          fill_in 'title', with: question_attributes[:title]
+          fill_in_react_ck 'textarea[name=description]', question_attributes[:description]
+          fill_in_react_ck 'textarea[name=staffOnlyComments]', question_attributes[:staff_only_comments]
+          fill_in 'maximumGrade', with: question_attributes[:maximum_grade]
+
+          find_field('Skills').click
+          find('li', text: skill.title).click
+
+          click_button 'Save changes'
+          sleep 0.2
+
+          question_created = assessment.questions.first.specific
+          expect(question_created.title).to eq(question_attributes[:title])
+          expect(question_created.description).to include(question_attributes[:description])
+          expect(question_created.staff_only_comments).to include(question_attributes[:staff_only_comments])
+          expect(question_created.maximum_grade).to eq(question_attributes[:maximum_grade])
+          expect(question_created.question_assessments.first.skills).to contain_exactly(skill)
+        end
+      end
+
+      scenario 'I can edit a question' do
+        voice = create(:course_assessment_question_voice_response, assessment: assessment)
+        visit course_assessment_path(course, assessment)
+
+        edit_path = edit_course_assessment_question_voice_response_path(course, assessment, voice)
+        find_link(nil, href: edit_path).click
+
+        maximum_grade = 999.9
+        fill_in 'maximumGrade', with: maximum_grade
```

Comment:
Could you change other fields as well (i.e title, description, etc)

Reply:
I have tried, but it can only be tested after PR.. so what I will do now shall be to commit then push it here.. hopefully all goes well